src/views/admin/PersonDetailsView.vue

```diff
@@ -163,6 +170,35 @@
             <v-col v-else-if="personStore.loading"> <v-progress-circular indeterminate></v-progress-circular></v-col>
           </v-row>
         </v-container>
+        <v-divider
+          class="border-opacity-100 rounded my-6 mx-4"
+          color="#E5EAEF"
+          thickness="6"
+        ></v-divider>
+        <!-- Zuordnungen -->
+        <v-container class="person-zuordnungen">
+          <v-row class="ml-md-16">
+            <v-col>
+              <h3 class="subtitle-1">{{ $t('person.zuordnungen') }}</h3>
+            </v-col>
+          </v-row>
+          <v-row>
+            <v-col
+              cols="10"
+              offset-md="2"
+              offset="1"
+              v-for="zuordnung in personenKontextStore.personenubersicht?.zuordnungen"
```

Comment:
```suggestion
              v-for="zuordnung in personenKontextStore.personenuebersicht?.zuordnungen"
```

Reply:
Soweit Ich weiss ja aber die Response vom Endpunkt ist ein Array von Zuordnungen und das wird auch möglich sein in der Zukunft.